docs/dev/dev_dagster.rst

```diff
@@ -0,0 +1,96 @@
+.. _dev_dagster:
+
+===============================================================================
+Developing with Dagster
+===============================================================================
+
+Reloading code locations
+------------------------
+
+When a new asset is added in the code it is not automatically added to the dag in
+dagit. To refresh the dag, click the small reload button next to the code location
+name in the top part of the UI:
+
+.. image:: ../images/reload_code_locations.png
+  :width: 800
+  :alt: Reload code locations
+
+Database schema changes
+-----------------------
+
+The :func:`pudl.io_managers.pudl_sqlite_io_manager` creates all
+required table schemas in the database when the ``pudl.sqlite``
+is originally created. The :func:`pudl.io_managers.pudl_sqlite_io_manager`
+replaces the existing data with the new data returned by the asset
+instead of droping the full table and its schema.
+
+This can lead to a scenario where a change in :mod:`pudl.metadata.resources`
+is not updated in the database. This will result in errors
+because the database and code are working with different
+versions of the metadata. **If you get an error that complains about
+:func:`pudl.io_managers.pudl_sqlite_io_manager` writing a column
```

Comment:
This link isn't working in the docs. Not sure if it's b/c of the bolding?

Reply:
The important takeaways you've bolded would make good [Note: or Warning:](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) boxes.

Reply:
This section is no longer relevant because of the work in #2523.